cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -43,6 +43,8 @@ import org.ossreviewtoolkit.utils.expandTilde
 import org.ossreviewtoolkit.utils.safeMkdirs
```

Comment:
Commit message: I think, the point here is that the available advisors are now stored in a map, so that they can be directly accessed by name. This should be expressed.